Detectors/MUON/MCH/Simulation/include/MCHSimulation/Digit.h

```diff
@@ -31,7 +31,7 @@ class Digit : public DigitBase
  public:
   Digit() = default;
 
-  Digit(int pad, double adc);
+  Digit(double time, int pad, double adc);
```

Comment:
I know this is not new, but I'm puzzled by the timestamp type. Why double  ? 
Probably more sth to see with Sandro, but I see there are two TimeStamp.h classes in O2 : 
- DataFormats/Headers/include/Headers/TimeStamp.h -> data oriented
- DataFormats/common/include/CommonDataFormat/TimeStamp.h -> used in all digitizers as far as I can tell.
The first one (data oriented) kind of makes sense to me as it's basically a long int. The second one, templatized, less so....